src/main/java/note/NoteList.java

```diff
@@ -239,7 +239,7 @@ public void openNoteProcess() throws InvalidNoteException {
                 while (!userInput.equals("")) {
                     if (!(userInput.startsWith("search") || userInput.startsWith("open"))) {
                         ui.printNoteErrorMessage(INVALID_NOTE_COMMAND_MESSAGE);
```

Comment:
If this method prints, after changes only contain INVALID_NOTE_COMMAND_MESSAGE, better to just inline the variable